ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -189,3 +208,83 @@ extension PillButtonBarDemoController: PillButtonBarDelegate {
         present(alert, animated: true)
     }
 }
+
+extension PillButtonBarDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        fluentTheme.register(tokenSetType: PillButtonTokenSet.self,
+                             tokenSet: isOverrideEnabled ? themeWideOverridePillButtonTokens : nil)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        self.bars.forEach({ bar in
+            let tokens = isOverrideEnabled ? perControlOverridePillButtonTokens : nil
+            bar.pillButtonOverrideTokens = tokens
+        })
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokens(for: PillButtonTokenSet.self) != nil
+    }
+
+    // MARK: - Custom tokens
+
+    private var themeWideOverridePillButtonTokens: [PillButtonTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .font: .uiFont {
+                return UIFont.fluent(FontInfo(name: "Times", size: 15.0, weight: .regular))
+            }
+        ]
+    }
+
+    private var perControlOverridePillButtonTokens: [PillButtonTokenSet.Tokens: ControlTokenValue] {
+        return [
+            .backgroundColor: .uiColor {
+                return UIColor(dynamicColor: DynamicColor(light: GlobalTokens.sharedColors(.steel, .tint40),
```

Comment:
Can't we simplify this to:
`UIColor(light: GlobalTokens.sharedColor(.steel, .tint40), dark: GlobalTokens.sharedColor(.steel, .shade30))`?